src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <DefineConstants>CORECLR</DefineConstants>
```

Comment:
Do you actually use any `#if CORECLR` code? I didn't see any in this review.

If not, you can remove this `DefineConstants` line.

If you want to keep this in, you should be doing it differently - it should be defined as 

```
<DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
```

Reply:
Ok I think we just copied it from somewhere, will remove.